content/posts/newsletter-010/index.md

```diff
@@ -566,6 +588,25 @@ _Discussions:
 [@lokathor]: https://twitter.com/lokathor
 [core::arch]: https://doc.rust-lang.org/nightly/core/core_arch/arch/index.html
 
+### [Point of WebGPU on Native][wgpu-post]
+
+![wgpu-rs logo](wgpu-logo.png)
+
+This month, Kvark shared their vision of
```

Comment:
Is this consistent with the other names used in the newsletter? I think the way I wrong gfx-rs entries before is by mentioning like `@kvark`